src/basic/MainBasic.java

```diff
@@ -0,0 +1,117 @@
+package basic;
+
+public class MainBasic {
+
+
+    public static void printListAllEmployees(Employee[] prt) {
+        System.out.println();
+        System.out.println("Output for Employee List start");
+        System.out.println("==================================");
+        for (int i = 0; i < prt.length; i++) {
+            System.out.println(prt[i]);}
+        System.out.println("==================================");
+        System.out.println("Output for Employee List finish");
+        System.out.println();
+    }
+
+    public static void printSalarySumAllEmployees(Employee[] slr) {
+        int totalSalaryAllEmployeesPerMonth = 0;
+        System.out.println();
+        System.out.print("Total salary per month for Employee List is: ");
+        for (int i = 0; i < slr.length; i++) {
+            totalSalaryAllEmployeesPerMonth = totalSalaryAllEmployeesPerMonth + slr[i].getSalary();
+        }
+        System.out.println(totalSalaryAllEmployeesPerMonth);
+        System.out.println();
+    }
+
+    public static void printSalaryMinimum(Employee[] min) {
+        int minimumSalaryPerMonth = min[0].getSalary();
+        System.out.println();
+        System.out.print("Minimum salary per month for Employee List is: ");
+        for (int i = 0; i < min.length; i++) {
+            if (min[i].getSalary() < minimumSalaryPerMonth) {
+                minimumSalaryPerMonth = min[i].getSalary();
+            }
+        }
+        System.out.println(minimumSalaryPerMonth);
+        System.out.println();
+    }
+
+    public static void printSalaryMaximum(Employee[] max) {
+        int maximumSalaryPerMonth = max[0].getSalary();
+        System.out.println();
+        System.out.print("Maximum salary per month for Employee List is: ");
+        for (int i = 0; i < max.length; i++) {
+            if (max[i].getSalary() > maximumSalaryPerMonth) {
+                maximumSalaryPerMonth = max[i].getSalary();
+            }
+        }
+        System.out.println(maximumSalaryPerMonth);
+        System.out.println();
+    }
+
+    public static void printSalaryAverage(Employee[] avg) {
+        int totalSalaryAllEmployeesPerMonth = 0;
+        int countOfEmployeesNonNull = 0;
+        System.out.println();
+        System.out.print("Average salary per month for Employee List is: ");
+        for (int i = 0; i < avg.length; i++) {
+            totalSalaryAllEmployeesPerMonth = totalSalaryAllEmployeesPerMonth + avg[i].getSalary();
+            if (avg[i].getSalary() > 0) {
```

Comment:
I believe 0 salary employees should be counted in the average calculation.
And if so you can just use `avg.length` in line 65